package.json

```diff
@@ -118,7 +118,7 @@
     "vitest": "1.6.0",
     "vue": "^3.2.37",
     "vue-router": "^4.0.16",
-    "vue-tsc": "^0.38.2"
+    "vue-tsc": "^1.8.27"
```

Comment:
I noticed that the version of `vue-tsc` has been updated to 2.x, and it is not possible to update to the new version here

Reply:
`typescript` need to be upgraded first cuz `vue-tsc@2.x` requires `typescript@5.x`. How about creating a new PR for it?

Reply:
ok